docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,199 @@
+- `camunda:asyncAfter`: Every task in Zeebe is always asyncBefore and asyncAfter.
+- `camunda:exclusive`: Jobs are always exclusive in Zeebe.
+- `camunda:jobPriority`: There is no way to prioritize jobs in Zeebe (yet).
+- `camunda:failedJobRetryTimeCycle`: You cannot yet configure the retry time cycle. Alternatively, you can [modify your code](/apis-tools/grpc.md#input-failjobrequest) to use the `retryBackOff` timeout (in ms) for the next retry.
+
+### Service tasks
+
+Migrating a service task is described in detail in the section about adjusting your source code below:
+
+![Service Task](../../components/modeler/bpmn/assets/bpmn-symbols/service-task.svg)
+
+A service task might have **attached Java code**. In this case, the following attributes/elements are migrated and put into a task header:
+
+- `camunda:class`
+- `camunda:delegateExpression`
+- `camunda:expression` and `camunda:resultVariable`
+
+The topic `camunda-7-adapter` is set.
+
+- `camunda:failedJobRetryTimeCycle`: Here, the amount of defined retries is set to the `zeebe:taskDefinition retries` attribute.
+
+A service task might leverage **external tasks** instead. In this case, the following attributes/elements are migrated:
+
+- `camunda:topic` becomes `zeebe:taskDefinition type`.
+
+The following attributes/elements **cannot** be migrated:
+
+- `camunda:taskPriority`
+
+Service tasks using `camunda:connector` will be migrated with the following changes:
+
+- `camunda:connectorId` becomes `zeebe:taskDefinition type`
+- All inputs and outputs are treated like all other inputs and outputs.
+
+### Send tasks
+
+![Send Task](../../components/modeler/bpmn/assets/bpmn-symbols/send-task.svg)
+
+In both engines, a send task has the same behavior as a service task. A send task is migrated exactly like a service task.
+
+### Gateways
+
+Gateways rarely need migration. The relevant configuration is mostly in the [expressions](/components/concepts/expressions.md) on outgoing sequence flows.
+
+### Expressions
+
+Expressions must be in [friendly-enough expression language (FEEL)](/components/concepts/expressions.md#the-expression-language) instead of [Java unified expression language (JUEL)](https://docs.camunda.org/manual/latest/user-guide/process-engine/expression-language/).
+
+Migrating simple expressions is doable (as you can see in [these test cases](https://github.com/camunda-community-hub/camunda-7-to-8-migration/blob/main/backend-diagram-converter/core/src/test/java/org/camunda/community/migration/converter/ExpressionTransformerTest.java)), but not all expressions can be automatically converted.
+
+The following is **not** possible:
+
+- Calling out to functional Java code using beans in expressions
+- Registering custom function definitions within the expression engine
+- Using SPIN library (the FEEL data structure behaves like JSON natively, so SPIN can be omitted or replaced)
+- Using the `execution` or `task`
+
+### Human tasks
```

Comment:
We flip between human tasks/user tasks -- can we unify on language here? I'm most familiar with usage of user task.

Reply:
https://github.com/camunda/developer-experience/issues/83